index.html

```diff
@@ -24,14 +24,205 @@ <h1 class="headline1">Hey There.<br>I'm Femi</h1>
+                    <p class="workgridp1">Profesional Art Printing Data/p>
+                    <p class="workgridp2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li><p>html</p></li>
+                        <li><p>bootstrap</p></li>
+                        <li><p>Ruby</p></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+                <div  class="workgrid4">
+                    <p class="workgridp1">Profesional Art Printing Data</p>
+                    <p class="workgridp2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li><p>html</p></li>
+                        <li><p>bootstrap</p></li>
+                        <li><p>Ruby</p></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="aboutme">
+            <div class="aboutmain">
+            <div class="aboutmain1">
+                <h2>About me</h2>
+                <p class="aboutp">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button type="button">Get My Resume</button>
+            </div>
+                <img class="abtimg" src="Images/Group1.png" alt="About image">
+                <img class="abtimg1" src="Images/Group(1).png" alt="About image">
+            </div>
+            <hr>
+            <div class="aboutsub">
+                <ul>
+                    <li class="sub1"><img src="Images/Rectangle55.png" alt="Shape"><p>Language</p></li>
+                    <li><p>Javascript</p></li>
+                    <li><p>Ruby</p></li>
+                    <li><p>Html</p></li>
+                    <li><p>Css</p></li>
+                </ul>
+                <ul>
+                    <li class="sub1"><img src="Images/Rectangle56.png" alt="Shape"><p>Frameworks</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby on Rails</p></li>
+                    <li><p>RSPec</p></li>
+                    <li><p>CapyBara</p></li>
+                    <li><p>Selenium</p></li>
+                </ul>
+                <ul>
+                    <li class="sub1"><img src="Images/Rectangle57.png" alt="Shape"><p>Skills</p></li>
+                    <li><p>Codekit</p></li>
+                    <li><p>Github</p></li>
+                    <li><p>Codepen</p></li>
+                    <li><p>Gitlab</p></li>
+                    <li><p>Terminal</p></li>
+                </ul>
+            </div>
+        </section>
```

Comment:
- [ ] Kindly make sure this section has similar margins (top, bottom, left, right) as defined in the design